apps/idos-enclave/src/features/recovery/backup.tsx

```diff
@@ -187,7 +186,7 @@ export function PasswordOrSecretReveal({
   );
 }
 
-export async function PasswordOrKeyBackup({
```

Comment:
I don't know what I was thinking about... this is a temporary fix, since in the new version on which I am working on, this will be solved in different way.